Bottom: b64771e0528836b1b08642ad8d65825f2b16eefb
Top:    51728d5521d1ee0b0549bbab8beaaac3072d54c0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 12:58:56 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
